csharp-intermediate-topics/TupleAliasInCSharp/TupleAliasInCSharp/TupleAliasInCSharp.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>    
```

Comment:
Ignore my previous comment. I added it before thinking about the fact that this code is using a new feature of .NET 8.0